js/google_ima.js

```diff
@@ -1844,9 +1844,10 @@ require("../html5-common/js/utils/utils.js");
      * @public
      * @method GoogleIMAVideoWrapper#setVideoUrl
      * @param {string} url The new url to insert into the video element's src attribute
+     * @param {string} encoding The encoding of video stream 
```

Comment:
This is fine; since you changed the comment in other places to include OO.VIDEO.ENCODING it could be done here as well but is not important.  Although actually since this field isn't used by IMA you don't need to include this at all; it should still work without this change.  Especially since IMA won't be able to use the encoding field based on its implementation